ARAnalytics.m

```diff
@@ -420,7 +433,7 @@ + (void)event:(NSString *)event {
 }
 
 + (void)event:(NSString *)event withProperties:(NSDictionary *)properties {
-    NSDictionary *combinedProperties = [_sharedAnalytics mergeProperties:properties forEvent:event];
+    NSDictionary *combinedProperties = [_sharedAnalytics mergeProperties:[ARAnalytics _combinePropertiesWithLocalizedHour:properties] forEvent:event];
```

Comment:
If there is only one property, maybe you can use a more simple method than merge?